src/lnd.rs

```diff
@@ -36,6 +35,8 @@ pub(crate) const MIN_LND_MINOR_VER: u32 = 18;
 pub(crate) const MIN_LND_PATCH_VER: u32 = 0;
 pub(crate) const MIN_LND_PRE_RELEASE_VER: &str = "beta";
 pub(crate) const BUILD_TAGS_REQUIRED: [&str; 3] = ["peersrpc", "signrpc", "walletrpc"];
+const SEED_KEY_FAMILY: u32 = 4;
+const SEED_KEY_INDEX: u32 = 4444;
```

Comment:
Are we able to leave comments on the choice of consts here? And should we just define them as `i32`s here anyway to avoid the cast below?

Reply:
there isn't logical reason in these constants. I'll add that. And yes, good catch on i32

Reply:
Ah yeah, thanks. I think the comment is good to mention it's just an arbitrary choice and no magic :)